rust/actyx/ax/tests/cli.rs

```diff
@@ -191,3 +196,76 @@ fn version() {
             .success();
     }
 }
+
+// calls `ax settings local [..args] --working-dir [working_dir]`
+fn call_cli_settings_local<I, S>(working_dir: impl Into<String>, args: I) -> Command
+where
+    I: IntoIterator<Item = S>,
+    S: AsRef<str>,
+{
+    let args = ["settings", "local"]
+        .into_iter()
+        .map(String::from)
+        .chain(args.into_iter().map(|x| String::from(x.as_ref())))
+        .chain([String::from("--working-dir"), working_dir.into()])
+        .collect::<Vec<_>>();
+
+    let mut proc = cli();
+    proc.args(args);
+    proc
+}
+
+#[test]
+fn cli_settings_local() {
+    let temp_dir = String::from(tempfile::tempdir().unwrap().path().to_str().unwrap());
+
+    // Init
+    {
+        // First --fail-on-existing init to empty dir should success
+        call_cli_settings_local(&temp_dir, ["init", "--fail-on-existing"])
+            .assert()
+            .success();
+        // Second --fail-on-existing init to the same dir should fail
+        call_cli_settings_local(&temp_dir, ["init", "--fail-on-existing"])
+            .assert()
+            .failure();
+        // Without --fail-on-existing, init to the same dir should success
+        call_cli_settings_local(&temp_dir, ["init"]).assert().success();
+    }
+
+    // Get/Set/Unset
+    {
+        let empty_dir = String::from(tempfile::tempdir().unwrap().path().to_str().unwrap());
```

Comment:
Could we separate these into singular tests or not really?

Reply:
Only for the empty_dir tests. The rest is kinda tied.